frontend/src/App.tsx

```diff
@@ -33,7 +37,6 @@ function App() {
               <Route path="blogs" element={<AdminBlogs />} />
             </Route>
           </Route>
-          <Route path="*" element={<NotFound />} />
         </Routes>
```

Comment:
Why is this removed? Can you please add it back

Reply:
ok sir